src/components/modal/index.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from 'react';
```

Comment:
@meysam-jeffrey Jan, if you possible add rule in webpack config that we don't need import react each file.

add plugins in webpack
```typescript
import { ProvidePlugin } from 'webpack';
...
 plugins: [
    new ProvidePlugin({
      React: 'react',
    }),
...
```

add these two lines in tsconfig:
```typescript
 "jsx": "react-jsx",
    "jsxImportSource": "react"
```

and finally add this section to the eslint.yml:
```yaml
plugins:
  - prettier
  - react
  - '@typescript-eslint'
rules:
  prettier/prettier:
    - error
  react/jsx-uses-react: 'off'
  react/jsx-uses-vars: 'warn'
  react/prop-types: 'off'
settings:
  react:
    version: detect

```